mstranslator.py

```diff
@@ -43,7 +44,8 @@ def __call__(self, r):
 
     def request_token(self):
         headers = {
-            'Ocp-Apim-Subscription-Key': self.subscription_key
+            'Ocp-Apim-Subscription-Key': self.subscription_key,
+            'Content-type': 'application/json'
```

Comment:
It's actually not required as we use `json=` parameter in requests. But if we keep it here, it would be better to leave a comma at the end of string:
`'Content-type': 'application/json',`

Reply:
Not sure about this, I'm not used to use Python, why a comma at the end?

Reply:
It's a good habit as python allows to use trailing comma for cleaner diffs. If there was a trailing comma at after `self.subscription_key` before, we would see only one line added (not one changed and one added).

Reply:
Ok, I see the point. I have just added.